### DISCUSSION TITLE
error - MFSU latest build error:  AssertionError [ERR_ASSERTION]: dependence not found: @umijs/plugin-antd/node_modules/antd/lib/style/themes/default.less

### DISCUSSION THREAD

**Discussion by Khadhiri786** [umijs/umi#11901] [General]
I'm getting error while upgrading umijs v3 to v4

**Comment by fz6m** [comment]
See https://github.com/umijs/umi/discussions/11900#discussioncomment-7654961 .

max (umi 4) includes all previous `@umijs/plugin-xxx` by default, no longer needed `@umijs/plugin-xxx`:

```ts
// umi 3

  plugins: ['@umijs/plugin-antd'],
  antd: {}
```

```ts
// max (umi 4, built-in all the `@umijs/plugins/dist/*` )

  antd: {}
```

```ts
// umi 4 ( required to install `@umijs/plugins` )

  plugins: ['@umijs/plugins/dist/antd'],
  antd: {}
```

`@umijs/max` is a collection of plugins, in contrast, umi 4 does not include any plugins.
